src/Assignments/ProblemIframe.tsx

```diff
@@ -119,7 +147,8 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
                 // TODO: impossi-log console.error()
                 setError(`No grades id for problem #${problem.id}`);
                 return;
-            }    
+            }
+            updateSubmitActive(_.isEqual(formDataToObject(formData), lastSubmission));
```

Comment:
Since it's part of save isn't there a 2 second delay?

Reply:
yes, after updating, the button will re-activate once the saveState is updated

Reply:
Let's add throttle